libs/blocks/how-to/how-to.js

```diff
@@ -47,7 +47,9 @@ const setJsonLd = (heading, description, mainImage, stepsLd) => {
 };
 
 const getImage = (el) => el.querySelector('picture') || el.querySelector('a[href$=".svg"');
-const getVideo = (el) => el.querySelector('video') || el.querySelector('.milo-video');
+const getVideo = (el) =>{
+  return el.querySelector('.video-container, .pause-play-wrapper, video, .milo-video');
+}
```

Comment:
This might cause a Lint error, since in such scenarios it prefers to return directly
```suggestion
const getVideo = (el) => el.querySelector('.video-container, .pause-play-wrapper, video, .milo-video');
```

Reply:
That's right, I used this syntax to debug while writing the functionality and forgot to change it back.